package.json

```diff
@@ -14,7 +14,8 @@
     "start": "PORT=3001 react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "cy:open": "yarn start --silent & cypress open"
```

Comment:
Under this line please add :
```
"cy:run": "BROWSER=none yarn start & cypress run"
```
this will be needed for Semaphore later, unless you deliberately waited with this until we get to the Semaphore-User story.